src/utilities.ts

```diff
@@ -141,3 +142,27 @@ export function runAnsibleCommands(outputChannel) {
             }
         })
 }
+
+var config: any = null;
+var extensionPath = "";
+
+export function setExtensionPath(path: string) {
+    extensionPath = path;
+}
+
+export function getConfigPath() {
+    return extensionPath + '/config.json';
```

Comment:
what's inside the config.json? is it ansible config, or this extension's config? if it's this extension's config, it's better to put it directly to vscode config.

Reply:
indeed you are right, i want to store things like:
- terminal launch string
- tbd
so i will create configuration option for terminal launch string

Reply:
how about put them into settings.json, instead of create a new one?